MetaMask/metamask-extension
# Removing storybook tests

## Explanation

* What is the current state of things and why does it need to change?
A while ago an effort was made to create unit tests for stories in an effort for the CI to flag if an update to the code had broken a storybook https://github.com/MetaMask/metamask-extension/pull/9840. Although this PR had the best of intentions it would have created a lot of overhead and not many storybook unit tests have been created. I'm also not sure if the action is still even being used during the build. There are also now better ways to smoke test storybook and add it to our CI processes which we will be implementing shortly. https://storybook.js.org/blog/interaction-testing-with-storybook/

* What is the solution your changes offer and how does it work?
This PR removes the changes originally made in https://github.com/MetaMask/metamask-extension/pull/9840 and https://github.com/MetaMask/metamask-extension/pull/12517

## Screenshots/Screencaps

### Before
Searching for storybook tests only one result

<img width="537" alt="Screen Shot 2022-11-17 at 6 53 22 PM" src="https://user-images.githubusercontent.com/8112138/202591510-642009bc-8979-428b-8dbc-703e2391732b.png">

### After
Searching for storybook tests now no results

<img width="518" alt="Screen Shot 2022-11-17 at 6 45 20 PM" src="https://user-images.githubusercontent.com/8112138/202591543-706ff197-9519-460c-ae35-61fe1b19973e.png">

Also no results for any storybook test actions

<img width="300" alt="Screen Shot 2022-11-17 at 6 46 35 PM" src="https://user-images.githubusercontent.com/8112138/202591706-4e611f93-427a-4d04-96fa-0801ca3ca7cd.png"><img width="300" alt="Screen Shot 2022-11-17 at 6 46 56 PM" src="https://user-images.githubusercontent.com/8112138/202591709-556b6746-7f1e-4234-bf1d-90df13eef0f2.png">

## Manual Testing Steps

- I think as long as the build passes this PR should be good to go
- Pull this branch and do a check to see if there are any other `.stories.test.js` files that I missed

## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
